src/HomeWork6/ClassesHomeWork6/Payment.java

```diff
@@ -0,0 +1,72 @@
+package HomeWork6.ClassesHomeWork6;
+
+public class Payment {
+    private int myMoney;
+    private boolean accept;
+    private PaymentSystem ps;
```

Comment:
better use clear name of variable, also if it is long name, not "ps" - "paymentSystem". We can use such name like ps for local variable